.github/workflows/poetry-lock-command.yml

```
@@ -0,0 +1,144 @@
+    - name: Append success comment
+      uses: peter-evans/create-or-update-comment@v4
+      if: steps.git-diff.outputs.changes == 'true'
+      with:
+        comment-id: ${{ steps.first-comment-action.outputs.comment-id }}
+        reactions: hooray
+        body: |
+          > ✅ `poetry lock` applied successfully.
+
+    - name: Append success comment (no-op)
+      uses: peter-evans/create-or-update-comment@v4
+      if: steps.git-diff.outputs.changes != 'true' && steps.git-diff-2.outputs.changes != 'true'
```

actionlint

property "git-diff-2" is not defined in object type {first-comment-action: {conclusion: string; outcome: string; outputs: {string => string}}; git-diff: {conclusion: string; outcome: string; outputs: {string => string}}; pr-info: {conclusion: string; outcome: string; outputs: {string => string}}; vars: {conclusion: string; outcome: string; outputs: {string => string}}}

(expression)